src/main/java/com/aquent/viewtools/GeoIP.java

```diff
@@ -91,6 +93,9 @@ public CityResponse getLocation(String ip) {
             try {
                 InetAddress ipAddress = InetAddress.getByName(ip);
                 response = cityLookup.city(ipAddress);
+            } catch (AddressNotFoundException e) {
```

Comment:
If we won't actually log this, do we need the empty loop to be there?

Reply:
I want to exclude this one exception from logging, the rest of the exceptions will be handled by the general catch.  I specifically don't want to do anything in this case, so yes, we need the empty block.